src/main/java/net/minecraftforge/fluids/capability/templates/EmptyFluidHandler.java

```diff
@@ -19,69 +19,47 @@
 
 package net.minecraftforge.fluids.capability.templates;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
 import net.minecraftforge.fluids.FluidStack;
-import net.minecraftforge.fluids.FluidTankInfo;
-import net.minecraftforge.fluids.capability.FluidTankProperties;
-import net.minecraftforge.fluids.IFluidTank;
-import net.minecraftforge.fluids.capability.IFluidTankProperties;
 import net.minecraftforge.fluids.capability.IFluidHandler;
 
-public class EmptyFluidHandler implements IFluidHandler, IFluidTank
+public class EmptyFluidHandler implements IFluidHandler
 {
     public static final EmptyFluidHandler INSTANCE = new EmptyFluidHandler();
-    public static final FluidTankInfo EMPTY_TANK_INFO = new FluidTankInfo(null, 0);
-    public static final IFluidTankProperties EMPTY_TANK_PROPERTIES = new FluidTankProperties(null, 0, false, false);
-    public static final IFluidTankProperties[] EMPTY_TANK_PROPERTIES_ARRAY = new IFluidTankProperties[] { EMPTY_TANK_PROPERTIES };
 
     protected EmptyFluidHandler() {}
 
     @Override
-    public IFluidTankProperties[] getTankProperties()
-    {
-        return EMPTY_TANK_PROPERTIES_ARRAY;
-    }
+    public int getTanks() { return 1; }
 
-    @Override
     @Nullable
-    public FluidStack getFluid()
-    {
-        return null;
-    }
-
     @Override
-    public int getFluidAmount()
-    {
-        return 0;
-    }
+    public FluidStack getFluidInTank(int tank) { return null; }
```

Comment:
What about a non-null policy similiar to ItemStack, with an isEmpty() chexk to find out, whether the FluidStack is empty?